scripts/date-logic/date-logic.js

```diff
@@ -54,5 +54,30 @@
     { '$count': 'numFormFields' }
   ]
   db.getCollection('forms').aggregate(afterScriptFormFieldsWithIsFutureOnlyTrue)
+ 
 
-  
\ No newline at end of file
+// REMOVE ISFUTUREONLY KEY 
```

Comment:
I'd suggest putting this script in a different file. Maybe `remove-date-logic`.

nit: please run the linter through this file, and then remove the dangling commas that appear haha

Reply:
done!